config/routes.rb

```diff
@@ -519,5 +519,9 @@
     end
   end
 
-  resources :notification_recipients, :only => [:index, :update, :destroy]
+  resources :notification_recipients, :only => [:index, :update, :destroy] do
+    collection do
+      put 'group/:id' => 'notification_recipients#update_group_as_read'
```

Comment:
Since the group is sent by name rather then ID, shouldn't this be `'group/:group` (and `params[:group]` in the controller)?